docs/layers/tile-3d-layer.md

```diff
@@ -118,6 +123,16 @@ Along with other options as below,
   - `url`: the url of the failed tile.
   - `message`: the error message.
 
+## Sub Layers
+
+The Tile3DLayer renders the following sublayers based on tile [format](https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification#introduction):
+
+* `scenegraph` - a [ScenegraphLayer](/docs/layers/scenegraph-layer.md) rendering all the tiles with Batched 3D Model format or Instanced 3D Model format.
+  - `_lighting` is default to `pbr`.
+* `pointcloud` - a [PointCloudLayer](/docs/layers/point-cloud-layer.md) rendering all the tiles with Point Cloud format.
+  - `pointSize` is default to `1.0`.
+  - `getColor` is default to `[255, 0, 0]`
```

Comment:
This is an unusual default color. It is only used if no color in the cloud right, maybe we need to state that?